egui_demo_lib/src/apps/demo/table_demo.rs

```diff
@@ -28,14 +30,30 @@ impl super::Demo for TableDemo {
 
 impl super::View for TableDemo {
     fn ui(&mut self, ui: &mut egui::Ui) {
-        ui.checkbox(&mut self.virtual_scroll, "Virtual scroll");
-        ui.checkbox(&mut self.resizable, "Resizable columns");
-
         // Leave room for the source code link after the table demo:
         StripBuilder::new(ui)
+            .size(Size::exact(66.0)) // for the settings
```

Comment:
You should be able to just put the settings before the `StripBuilder`. We only need `StripBuilder` here because of the table uses `remainder`.

Reply:
I don't know any other way to get a horizontal layout (checkbox options on the left, slider on the right). If I move the setting strip builder with the horizontal layout outside of the top-level strip builder than the whole table demo window automatically expands in height well below the bottom edge of the application window.

Anyway, if this is a demo what's the problem with demonstrating how to embed a horizontally-oriented strip inside the cell of a vertically-oriented strip?